macosx/Torrent.mm

```diff
@@ -863,8 +863,13 @@ - (CGFloat)availableDesired
 
 - (BOOL)isActive
 {
-    return self.fStat->activity != TR_STATUS_STOPPED && self.fStat->activity != TR_STATUS_DOWNLOAD_WAIT &&
-        self.fStat->activity != TR_STATUS_SEED_WAIT;
+    return self.fStat->peersSendingToUs > 0 || self.fStat->peersGettingFromUs > 0 || self.fStat->webseedsSendingToUs > 0 ||
+        self.fStat->activity == TR_STATUS_CHECK;
+}
+
+- (BOOL)isPaused
+{
+    return self.fStat->activity == TR_STATUS_STOPPED;
 }
```

Comment:
If `isPaused` is the negative version of what was previously defined as `isActive`, then shouldn't it be exactly:
```
return self.fStat->activity == TR_STATUS_STOPPED || self.fStat->activity == TR_STATUS_DOWNLOAD_WAIT ||
         self.fStat->activity == TR_STATUS_SEED_WAIT;
```